Core/IServiceModule.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using AeroGear.Mobile.Core;
+using AeroGear.Mobile.Core.Configuration;
+
+namespace AeroGear.Mobile.Core
+{
+    public interface IServiceModule
+    {
+     
+        /// <summary>
+        /// Type/name used in the mobile-singleThreadService.json
+        /// </summary>
+        /// <value>the type/name used to identify the singleThreadService config in the JSON file</value>
```

Comment:
what does `singleThreadService` mean here?

Reply:
yes, that's nonsense from some replace...